core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/BlueprintV4Controller.java

```diff
@@ -68,6 +68,13 @@ public BlueprintV4Response getByName(Long workspaceId, @ResourceName @NotNull St
         return converterUtil.convert(blueprint, BlueprintV4Response.class);
     }
 
+    @Override
+    @InternalOnly
+    public BlueprintV4Response getByNameInternal(Long workspaceId, @AccountId String accountId, @NotNull String name) {
+        Blueprint blueprint = blueprintService.getByWorkspace(NameOrCrn.ofName(name), restRequestThreadLocalService.getRequestedWorkspaceId());
+        return converterUtil.convert(blueprint, BlueprintV4Response.class);
+    }
+
```

Comment:
this is the same logic as in `getByName`, can you reuse that?

Reply:
Ohh, sh.. I was blind. Sure.

Reply:
it's not a big deal:) thx!